### PR TITLE
Adventurer Skills Re-scope

### PR DESCRIPTION
**Pull request by LynxSolstice**
## About The Pull Request

Modifies the adventurer skills so each class is no longer a jack of all trades, but rather specialized to their specific classes.

## Why It's Good For The Game
Overgeneralization of the many archetypes waters them all down into slightly different flavors of "Combat class but it can craft, and maybe has one or two extras over the other classes" rather than truly being different archetypes to choose from.

### CODE REVIEW

**Comment by Wesreidau** [comment]
Letting players pick a weapon or aesthetic of choice, or the barbarian etc having some outdoors skills, made them versatile for different character types and gave them roleplaying breadth as opposed to just being a combat stat block for one or two specific weapons.

**Comment by Troopermoores** [comment]
Kill this with fire, if you want to make people play townie roles don't nerf what people play already, make fucking townie fun

**Comment by Wesreidau** [comment]
I'm sorry, "Skill bloat is bad" is not an argument, its an assertion. You'd have to make a case against player freedom. Adventurers had a rounded skillset of novice and apprentice talents to help them survive while adventuring. It has nothing to do with making them not need to interact with the town, as the armorsmith line proves.

There is a reason DnD had a broad concept of 'martial' weapons; players want to create their own character armed with what they like them to have, not fit into a cookie-cutter typeset. Those roles belong elsewhere in regimented soldiery like the watch footman or archer subclasses, where you take away player freedom and customization in exchange for power.

I do not want to see a change of this nature go through, where Rangers lose all their 'ranging' skills and become just a combat class with a bow, or barbarians are told to grind their polearm skill with a knight if they want to use a bardiche and not a battle axe. It is nonsensical to encourage more grinding the t0 crafting recipes so a ranger knows how to build a drying rack.

You asked me for constructive feedback and the best advice I can give is to recognize this might match your vision, but I doubt the rest of the players are looking for this. Going on for as long as I could against this will only seem spiteful.

**Comment by DeltaFire15** [comment]
After the discussion on the discord, I have to agree that this is overboard.

This cuts most non-combat focused skills from Adventurers, then also decides to cut their combat skills, too.

The thing is, an Adventurer's Adventure is not just "24/7 kill monsters". See, things like D&D or other TTRPGs where a lot of the time, your non-combat skills are actually in the focus.
In this game, your character may just have spawned, but they have already lived for several years, surely not only exclusively learning how to swing a sword during their travels.

Then you have this, where, as a very obvious example, a Ranger, who breathes the wilds, cannot butcher nor cook, nor has ever crafted anything? There's several such examples, but I feel like Ranger is one of the most obvious ones where what is a "Utility wilds gremlin" is cut down to "You are a guy with a bow."

Also, this cuts reading from many classes, which goes against the previous change a while ago that intentionally gave almost every class reading (#59 ) so writing material had more use.



Townies should be better at crafting and the like because they *have* just been living in a town and most likely practicing their craft, but that does not make Adventurers bumbling fools. Just saying "they can just grind those skills again" is not a good justification for this, either, as tedium is not good gameplay and that time could be spent on creating stories instead.

tl;dr; Not exclusively knowing how to slap things is not "bloat".

**Comment by Stutternov** [comment]
How can one PR be so based?

**Comment by Ephemeralis** [comment]
It's difficult to differentiate 'some crafting proficiency' versus 'can make practically everything' with the way skills are currently set up.

I think this PR is broadly positive - adventurers need to definitely be less _do everything_, and many of the skillpacks they previously received were sort of ridiculous.

What might be worth addressing is adding an extra suite of "favored skills" to a job/role that have an XP buff and can be advanced past journeyman. A bit beyond the scope of this PR though.

**Comment by Stutternov** [comment]
On a serious note -

I wouldn't give ranger that level of skills in crafting. Even level 1 tanning is generous. Unlike rogue they have medium armor training and dodge expert.

On top of that - Hunter, a towner role, is actually pretty decent but those stats makes Hunter useless as Ranger is actually better at crafting than even Hunter is. 

As for Barb/Amazon - 2 tanning is a bit too generous, if they get any tanning at all it should be 1 probably.

Keep in mind - crafting skill merely lets you craft stuff at that level and increasing chances of crafting prior levels. But intelligence can give you a slight bypass and you grind levels really quick normally.
